templates/books-show.html

```diff
@@ -0,0 +1,39 @@
+<div class="container">
+  <div class="row">
+    <div class="col-md-10 col-md-offset-1">
+      <div class="panel panel-default">
+        <div class="panel-body">
+          <ul class="list-group form-inline">
+            <div class="row">
```

Comment:
The fact that you have rows within rows here smells like copy-pasted code that wasn't edited; it seems like you have WAY more outer divs than you need.